### PR TITLE
Add Riemann Quantum Classifier 

### PR DESCRIPTION
**Pull request by gcattan**
This PR implements #23.
The helper classifier which is implemented here will be used for CVGridSearch.
A couple of tests were added to `test_classification.py` and required to refactorize some of the methods into a dataset/utils module.

(WIP: missing documentation of the dataset module, and error in the pipeline)

### CODE REVIEW

**Review by qbarthelemy — CHANGES_REQUESTED** [review]

File: pyriemann_qiskit/classification.py [hunk 1]
```diff
@@ -417,3 +422,192 @@ def predict(self, X):
         """
         _, labels = self._predict(X)
         return self._map_0_1_to_classes(labels)
+
+
+class RiemannQuantumClassifier(BaseEstimator, ClassifierMixin,
```
[the remark this answers is not in the document]
Comment (gcattan) [reply pyriemann_qiskit/classification.py:427]: I agree. I will try to find something more appropriate.

**Comment by gcattan** [comment]
Thanks for the review!

There is still a problem with the pipeline, using python 3.8. To be honest I am not sure where it comes from.
Tests are ok on my local computer.

**Comment by gcattan** [comment]
Ok, I compared the package version with older builds and it seems that the scipy version changed 2 days ago. 

@toncho11 FYI

**Review by qbarthelemy — APPROVED** [review]

**Comment by gcattan** [comment]
Thanks for the review :)

**Pull request merged**